code/modules/mob/living/carbon/human/human_helpers.dm

```diff
@@ -156,3 +156,14 @@
 			return FALSE
 
 	return .
+
+/mob/living/carbon/human/proc/remove_catshit()
+	var/obj/item/organ/ears/cat/E = getorganslot(ORGAN_SLOT_EARS)

```

Comment:
What if they have non cat mutant ears?

Reply:
won't happen, but if it does then the policy will still be that they have to be fully human to be a head anyway